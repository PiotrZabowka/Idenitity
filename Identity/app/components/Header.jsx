import styled from 'styled-components';

export default styled.h1`
    font-weight: bolder;
    background-color: red;
`;
