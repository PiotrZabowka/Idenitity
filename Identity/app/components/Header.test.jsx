import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <Header>Facebook</Header>,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
