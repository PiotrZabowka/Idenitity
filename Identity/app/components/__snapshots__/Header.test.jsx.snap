exports[`test Link changes the class when hovered 1`] = `
<h1
  className="hrfEcl">
  Facebook
</h1>
`;
